odl/set/pspace.py

```diff
@@ -619,10 +619,34 @@ def __repr__(self):
 
         return '{!r}.element({})'.format(self.space, inner_str)
 
-    def show(self, *args, **kwargs):
-        """Display the parts of this vector."""
+    def show(self, indices=None, **kwargs):
+        """Display the parts of this vector graphically
+
+        Parameters
+        ----------
+        indices : index expression, optional
+            If a single index (``indices=0``) displays that part
+            If a slice, displays those parts (``indices=slice(None)``)
+            If `list` of indices, display those (``indices=[0, 1, 3]``)
+            If a tuple is given, interprets the first element as an index for
+            this product space vector and passes the rest on.
+            (``indices=np.s_[0, :, :]``).
```

Comment:
Somewhat bumpy language here, and I don't really understand what is being said here. From the code it appears to me that lists and tuples are equivalent.
It would be good to clarify what is differentiated between at all. For example like this 
```
Indices can refer to parts of a `ProductSpaceVector` and slices
in the parts in the following way:

Single index (``indices=0``) => display that part
Single slice (``indices=slice(None)``) or index list (``indices=[0, 1, 3]``) => display those parts
Sequence of slices  (``indices=np.s_[0, :, :]``): take the first axis to select the parts
and pass the rest on.
```